### PR TITLE
Changing the Web SDK version to 2.1.0-beta1 unstable, referencing Core SDK of the same unstable version.

### PR DESCRIPTION
**Pull request by tokaplan**
Changing the Web SDK version to 2.1.0-beta1, referencing Core SDK of the same version.

### CODE REVIEW

File: Src/WindowsServer/WindowsServer.Shared.Tests/WindowsServer.Shared.Tests.shproj [hunk 1]
```diff
@@ -7,6 +7,9 @@
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.Default.props" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.props" />
   <PropertyGroup />
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (abaranch) [inline Src/WindowsServer/WindowsServer.Shared.Tests/WindowsServer.Shared.Tests.shproj]: What's this?

File: Test/PerformanceCollector/FunctionalTests/TestApps/TestApp40/Web.config [hunk 2]
```diff
@@ -86,6 +86,10 @@
         <assemblyIdentity name="Microsoft.Diagnostics.Tracing.EventSource" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-1.1.28.0" newVersion="1.1.28.0" />
       </dependentAssembly>
+      <dependentAssembly>
```
Comment (abaranch) [inline Test/PerformanceCollector/FunctionalTests/TestApps/TestApp40/Web.config]: You should not need this....

File: Test/PerformanceCollector/FunctionalTests/TestApps/TestApp45/Web.config [hunk 3]
```diff
@@ -62,6 +62,10 @@
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.ApplicationInsights" publicKeyToken="31bf3856ad364e35" culture="neutral" />
```
Comment (abaranch) [inline Test/PerformanceCollector/FunctionalTests/TestApps/TestApp45/Web.config]: Here as well

File: Src/WindowsServer/WindowsServer.Shared.Tests/WindowsServer.Shared.Tests.shproj (hunk 1 above)
Comment (tokaplan) [inline Src/WindowsServer/WindowsServer.Shared.Tests/WindowsServer.Shared.Tests.shproj]: No clue, removed.

File: Test/PerformanceCollector/FunctionalTests/TestApps/TestApp40/Web.config (hunk 2 above)
Comment (tokaplan) [inline Test/PerformanceCollector/FunctionalTests/TestApps/TestApp40/Web.config]: Removed.

File: Test/PerformanceCollector/FunctionalTests/TestApps/TestApp45/Web.config (hunk 3 above)
Comment (tokaplan) [inline Test/PerformanceCollector/FunctionalTests/TestApps/TestApp45/Web.config]: Removed.

**Comment by abaranch** [comment]
:shipit:

**Pull request merged**